result/array-bracket-spacing.js

```diff
@@ -0,0 +1,118 @@
+/*eslint-env es6*/
+
+var arr = [ 'foo', 'bar' ];
+var [ x, y ] = z;
```

Comment:
__[development][eslint]__
* Error - 'x' is defined but never used (no-unused-vars)
* Error - 'y' is defined but never used (no-unused-vars)
* Error - 'z' is not defined. (no-undef)